{
  "schema_version": "1.4.0",
  "id": "GHSA-53v9-6jr7-7fxh",
  "modified": "2024-10-10T15:30:41Z",
  "published": "2024-10-10T15:30:41Z",
  "aliases": [
    "CVE-2024-35202"
  ],
  "details": "Bitcoin Core before 25.0 allows remote attackers to cause a denial of service (blocktxn message-handling assertion and node exit) by including transactions in a blocktxn message that are not committed to in a block's merkle root. FillBlock can be called twice for one PartiallyDownloadedBlock instance.",
  "severity": [

  ],
  "affected": [

  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2024-35202"
    },
    {
      "type": "WEB",
      "url": "https://github.com/bitcoin/bitcoin/pull/26898"
    },
    {
      "type": "WEB",
      "url": "https://bitcoincore.org/en/2024/10/08/disclose-blocktxn-crash"
    },
    {
      "type": "WEB",
      "url": "https://en.bitcoin.it/wiki/Common_Vulnerabilities_and_Exposures"
    },
    {
      "type": "WEB",
      "url": "https://github.com/bitcoin/bitcoin/blob/master/doc/release-notes/release-notes-25.0.md"
    },
    {
      "type": "WEB",
      "url": "https://github.com/bitcoin/bitcoin/releases/tag/v25.0"
    }
  ],
  "database_specific": {
    "cwe_ids": [

    ],
    "severity": null,
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2024-10-10T13:15:14Z"
  }
}